### PR TITLE
veusz: Fix `pre_install` script error. Also fix `checkver`

### LINKED ISSUE #9483: [veusz] pre_install script failed
## Bug Report

**Package Name:** [veusz]

### Current Behaviour

https://github.com/ScoopInstaller/Extras/blob/4403709016eb75408857cc6d8d3059335e468a36/bucket/veusz.json#L12

which cause `Remove-Item: A positional parameter cannot be found that accepts argument`

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #9483

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/veusz.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
         ]
     ],
     "checkver": {
-        "github": "https://github.com/veusz/veusz",
-        "regex": "download/veusz-([\\d.]+)"
+        "url": "https://github.com/veusz/veusz/releases/latest",
```
Comment (HUMORCE) [inline bucket/veusz.json:21]:
```suggestion
        "github": "https://github.com/veusz/veusz",
```

File: bucket/veusz.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
             "hash": "7e47e9d08e9393ec1a8d786716d5e11cc93ac7a10a8164e7490736c7a2960dc4"
         }
     },
-    "pre_install": "Remove-Item \"$dir\\`$*\" \"$dir\\uninst*\" -Force -Recurse",
+    "pre_install": "Remove-Item @(\"$dir\\`$*\", \"$dir\\uninst*\") -Force -Recurse",
```
Comment (HUMORCE) [inline bucket/veusz.json:12]:
```suggestion
    "pre_install": "Remove-Item \"$dir\\`$*\",\"$dir\\uninst*\" -Force -Recurse",
```

**Review by HUMORCE — CHANGES_REQUESTED** [review]

File: bucket/veusz.json (hunk 1 above)
Comment (Zliced13) [reply bucket/veusz.json:21]: I tried using "GitHub" in checkver, but that resulted in an error for me.

File: bucket/veusz.json (hunk 2 above)
Comment (Zliced13) [reply bucket/veusz.json:12]: The array is necessary, or else the Remove-Item command will not work properly.

Comment (HUMORCE) [reply bucket/veusz.json:12]: the original line not compatible with powershell 5.1, but separating them with comma is works, tested on powershell 5.1/core LTS 7.2.6.

Comment (Zliced13) [reply bucket/veusz.json:12]: oh ok, I'll change it.

File: bucket/veusz.json (hunk 1 above)
Comment (HUMORCE) [reply bucket/veusz.json:21]: they are equivalent 
https://github.com/ScoopInstaller/Scoop/blob/8aee6f99803d0f0aeab252e1d9d4848777185c51/bin/checkver.ps1#L145-L164

screenshot_20221018055238

Comment (Zliced13) [reply bucket/veusz.json:21]: OK if that is the same, I'll use that instead.

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**